actionmailer/test/message_delivery_test.rb

```diff
@@ -57,7 +57,7 @@ class MessageDeliveryTest < ActiveSupport::TestCase
     assert_respond_to @mail, :deliver_now!
   end
 
-  def test_should_enqueue_and_run_correctly_in_activejob
+  test "should enqueue and run correctly in activejob" do
     @mail.deliver_later!
     assert_equal 1, ActionMailer::Base.deliveries.size
   end
```

Comment:
I think we could just remove this test instead.  It looks like the described behavior is thoroughly covered by the next two tests, right?

https://github.com/rails/rails/blob/ef04fbb3b256beececfa44c47c4ec93ac6945e59/actionmailer/test/message_delivery_test.rb#L65-L75